charts/seed-bootstrap/charts/vpa/charts/application/templates/clusterrole-actor.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if or .Values.recommender.enabled .Values.updater.enabled }}
```

Comment:
Somehow missed this change in my last review because in some of the first few round of reviews/commits, this check was not there.
Curious as to why do we need to check if recommender, updater and admission-controller are enabled separately. is it ok to merge them under single umbrella of `vpa.enabled`, or can there be a scenario when we deploy them partially?

Reply:
@wyb1 suggested to not deploy the vpa-exporter when VPA is enabled for shoots. That's why I also made is configurable for the other components (to be consistent), although, it's currently only used to enable/disable vpa-exproter.

Reply:
ok. thanks for explaining. :)